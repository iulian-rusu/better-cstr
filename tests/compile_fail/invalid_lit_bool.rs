use better_cstr::c;

fn main() {
    c!(true);
}
