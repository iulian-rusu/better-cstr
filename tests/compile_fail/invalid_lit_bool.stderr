error: Unsupported literal kind
 --> tests/compile_fail/invalid_lit_bool.rs:4:8
  |
4 |     c!(true);
  |        ^^^^
